Update homework 1

jlperona opened the pull request:
If you want to separate out the released files versus the future versions, you can still accept this PR and I can go back into the version history to dig out the file before I made changes

jlperona committed:
Expand on the platforms, add submission details

Thankfully, if we want to move to a master version of the homework
files, there's a clear split in the commit history

jlperona committed:
Fix spelling mistakes

Also make gcc reference consistent

powerjg commented on hw1/hw1.rst:
```diff
@@ -32,42 +32,46 @@ If you have any feedback about errors, big or small, please let me know!
 
-Go through the Introduction and Part I of the `gem5 tutorial`__.
-You will go through the following topics:
+Go through the Introduction and Building gem5 pages of the `gem5 tutorial`__.
```
We can do this now, in a later change, or I can do it next time I teach this, but I'd like rename tutorial->book everywhere.

jlperona replied:
I'll do it as part of this PR.

jlperona replied:
Resolved.

powerjg reviewed (commented):
Thanks for cleaning all of this up. Let me know how you want to deal with the issue raised below. I'm fine with merging this as-is, if you think that's easiest.

jlperona committed:
Merge branch 'master' of github.com:jlpteaching/17fq-cs201a into hw1

jlperona committed:
References to tutorial changed to book

I left the two where it made sense. Also, you may want to do a run through your book for the same consistency check.

powerjg commented:
Thanks!
